utils/load_utils.py

```diff
@@ -0,0 +1,377 @@
+"""
+Loading data helper functions
+"""
+
+import pathlib
+import pandas as pd
+
+from .data_utils import process_tissue_ccle_column, process_missing_tissues
+
+# Set constants
+# Note the dictionary key is both the dataset and also the directory
+FILENAMES = {
+    "ccle": {"viability": "CCLE_NP24.2009_Drug_data_2015.02.24.csv"},
+    "depmap": {
+        "viability": "CRISPRGeneDependency.csv",
+        "celllines": "Model.csv",
+        "genes": "depmap_gene_dictionary.tsv",
+    },
+    "prism": {
+        "primary_screen": {
+            "viability": "primary-screen-replicate-collapsed-logfold-change.csv",
+            "celllines": "primary-screen-cell-line-info.csv",
+            "treatments": "primary-screen-replicate-collapsed-treatment-info.csv",
+        },
+        "secondary_screen": {
+            "viability": "secondary-screen-replicate-collapsed-logfold-change.csv",
+            "celllines": "secondary-screen-cell-line-info.csv",
+            "treatments": "secondary-screen-replicate-collapsed-treatment-info.csv",
+        },
+    },
+    "nci60": {"viability": "DOSERESP.csv", "treatments": "chemnames_Aug2013.zip"},
+}
+
+CCLE_MISSING_TISSUE_NAMES = {
+    "M059J": "GLIAL",
+    "SF8657": "FIBROBLAST",
+    "SNUC2B": "LARGE_INTESTINE",
+}
+
+DEPMAP_MISSING_TISSUE_NAMES = {
+    "NCC-LMS1-C1": "BONE",
+    "NCC-MPNST2-C1": "MPNST",
+    "A375_RPMI": "SKIN",
+    "RVH421_RPMI": "SKIN",
+}
+
+
+def load_ccle(
+    top_dir, data_dir="ccle/data", process_tissues=False, only_get_cells=False
+):
+    """
+    Load Cancer Cell Line Encyclopedia (CCLE) viability data
+
+    Arguments:
+    ----------
+    top_dir : str
+        Where the repository top level directory is in relation to your current wd
```

Comment:
Seeing how `top_dir` is specified and used below, consider using a structure for this repo to enable consistent execution without relative pathing for the data in the function. Doing this might save time and simplify the use of data throughout. One example of how to do this might entail always running things from a single location. I bring this up to help spur thought around this - this isn't a requirement for review.

Reply:
> always running things from a single location.

My intention is for this function to be run in multiple locations (or at least to not require a strict location). I think for a data analysis repository this level of flexibility and potential for user error is ok